doc/releases/changelog-0.21.0.md

```diff
@@ -280,16 +273,18 @@
   The adjoint method requires memory for 4 independent state vectors, which corresponds roughly
   to storing a state vector of a system with 2 additional qubits.
 
-* A new method `qml.gradients.param_shift_hessian` has been added to directly compute the Hessian
-  (2nd order partial derivative matrix) of QNodes and QuantumTapes. The method generates
+<h4>New parameter-shift Hessian function</h4>
```

Comment:
if we decide to keep a separate subheading here, I would also suggest making it more descriptive, something like
```suggestion
<h4>Compute the Hessian on hardware</h4>
```

Reply:
Changed to have this.